app/src/main/java/com/github/se/cyrcle/MainActivity.kt

```diff
@@ -43,18 +45,26 @@ class MainActivity : ComponentActivity() {
 
   @Inject lateinit var authenticator: Authenticator
 
+  private val reviewViewModel: ReviewViewModel by viewModels {
+    CustomViewModelFactory { ReviewViewModel(reviewRepository) }
+  }
+  private val userViewModel: UserViewModel by viewModels {
+    CustomViewModelFactory { UserViewModel(userRepository, parkingRepository) }
```

Comment:
you removed a new argument I just added to the userviewmodel which is the imageRepository,

It still compiles because I added a default null arg. but I can't log in anymore 

<img width="606" alt="image" src="https://github.com/user-attachments/assets/eed5d398-a0b2-428b-afc2-7ba99f879be6">
Easy to debug thanks to the log

Reply:
Okay that maybe also why the CI don't work thanks i will look into it right away.

Reply:
I think I need to rebase i branched before your PR.